components/mcp230xx.rst

```diff
@@ -0,0 +1,75 @@
+MCP230xx I/O Expander
+=====================
+
+.. seo::
+    :description: Instructions for setting up MCP23017 or MCP23008 digital port expanders in ESPHome.
+    :image: mcp230xx.png
+
```

Comment:
Add a short paragraph at the top about how this page is for both MCP23017 and MCP23008.

Then make MCP23017 and 008 into their own "sections" (--- underline)

Reply:
Done.